src/Command.ino

```diff
@@ -454,6 +455,19 @@ void ExecuteCommand(byte source, const char *Line)
     break;
   }
 
+  case cmd_TaskValueSetAndRun:
+  {
+    success = true;
+    if (GetArgv(Line, TmpStr1, 4))
+    {
+      float result = 0;
+      Calculate(TmpStr1, &result);
+      UserVar[(VARS_PER_TASK * (Par1 - 1)) + Par2 - 1] = result;
+      SensorSendTask(Par1 - 1);
```

Comment:
Are you sure about all those "-1" ?

Reply:
Yes I am sure. The tasks commands (taskvalueset and taskrun) start from 1 while internally it starts from 0. Thus the minus 1.
My TaskValueSetAndRun is the merge of TaskValueSet + TaskRun.